kv/txn_coord_sender.go

```diff
@@ -788,28 +587,22 @@ func (tc *TxnCoordSender) heartbeat(id string, trace *tracer.Trace, ctx context.
 		return false
 	}
 
-	request := &proto.HeartbeatTxnRequest{
-		RequestHeader: proto.RequestHeader{
-			Key: txn.Key,
-			Txn: &txn,
-		},
-	}
-
-	request.Header().Timestamp = tc.clock.Now()
-	reply := &proto.HeartbeatTxnResponse{}
-	call := proto.Call{
-		Args:  request,
-		Reply: reply,
-	}
+	hb := &proto.HeartbeatTxnRequest{}
```

Comment:
why not use the struct literal syntax that was here before? this makes the diff needlessly harder to review